static/lang/en.json

```diff
@@ -5600,6 +5594,12 @@
                     "success": "Sonic damage dealt by this weapon ignores the target's sonic resistance."
                 }
             },
+            "greaterDisrupting": {
+                "Name": "Greater Vitalizing",
+                "Note": {
+                    "criticalSuccess": "On a critical hit, the undead creature is @UUID[Compendium.pf2e.conditionitems.Item.MIRkyAjyBeXivMa7]{Enfeebled 1} and @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1} as long as it has the persistent damage from this rune."
+                }
+            },
```

Comment:
?

Reply:
Oh

That section got moved. Hold on. Let me fix this.

Reply:
That part is reverted. Should no longer ping as a change.